lib/js_render/console_replay.js

```diff
@@ -0,0 +1,19 @@
+(function(history) {
+  var result = '';
+  if (history && history.length > 0) {
+    result += '\n<scr'+'ipt class="js_render-console-replay">';
```

Comment:
whats the reason for breaking up the script tag string?

Reply:
I cargo culted myself.